src/main/webapp/app/exam/participate/exam-participation.component.ts

```diff
@@ -888,4 +889,15 @@ export class ExamParticipationComponent implements OnInit, OnDestroy, ComponentC
                 }
             });
     }
+
+    asQuizExercise(exercise: Exercise): QuizExercise | undefined {
```

Comment:
this does not really look like good code to be honest

Reply:
This is removed since the input was changed to the QuizConfiguration interface.